<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="berdiri">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Berdiri</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lambaikan">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Dadah..</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kuda2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Kuda kuda</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pukul">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>60</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Pukul</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bonus">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>110</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Bonus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="jalan">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Jalan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="senamTgn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Senam tgn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="jalanTempat">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Jalan tetap</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gerakan Terkini    :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSetting">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>APA hayo</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_ports">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>79</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_mpuData">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QQuickWidget" name="quickWidget3D">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>300</width>
     <height>200</height>
    </rect>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::ResizeMode::SizeRootObjectToView</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
